### DISCUSSION TITLE
Catboost in base environment

### DISCUSSION THREAD

**Discussion by shaido987** [PaddlePaddle/PaddleSpatial#124] [BaiduKDDCup2022] [answered]
Hello,

Would it be possible to add catboost to the base environment?

github: https://github.com/catboost/catboost
pip install: https://catboost.ai/en/docs/installation/python-installation-method-pip-install

**Comment by ramber1836** [comment, marked as answer]
This package has already been added to the base environment.
Please refer to the latest version of the Manual on Evaluation, and the list of supported packages will be updated from time to time.
